distribution/changelog.txt

```diff
@@ -8,6 +8,7 @@
 - Fix: [#10228] Can't import RCT1 Deluxe from Steam.
 - Fix: [#10325] Crash when banners have no text.
 - Fix: [#10420] Money effect causing false positive desync.
+- Fix: [#10477] Multitile Scenery unable to be placed higher using shift.
```

Comment:
```suggestion
- Fix: [#10477] Large scenery cannot be placed higher using SHIFT.
```

Reply:
Any particular reason for capitalising 'shift' as such?

Reply:
As it's meaning the SHIFT key

Reply:
Of course, but I've never seen it referred to in text in all-caps. Just the initial letter, usually.

Reply:
We refer to it in ALL CAPS in our shortcut keys dialog, too.